packages/xarc-app-dev/src/config/opt2/webpack-options.ts

```diff
@@ -55,11 +55,23 @@ export type WebpackOptions = {
   devArtifactsPath?: string;
 
   /**
-   * Set to true to explicitly enable CSS module support for all style files
-   * - **Default: `undefined` (auto detect)**
-   * - If not set, then check env `CSS_MODULE_SUPPORT`
+   * Configure CSS module support.
+   *
+   * Settings:
+   * 1. `true` or "cssOnly" (**default**): CSS module for any file with `.css` extension
+   * 2. RegExp: CSS module for any file that matches the custom RegExp
+   * 3. `"all"`: CSS module for all style files (`css`, `styl`, `sass`, `scss`)
+   * 4. `"byModExt"`: Use the preset RegExp that matches any file ending with `.mod.{ext}` or `.module.{ext}`
```

Comment:
Just wondering if this name will be confusing. Since we are enabling based on file name, should we change this to `byFileName` or something? `byModExt` makes more sense if we let users specify the list of extensions.

Reply:
the check is always by filename through the regexp.  in this case, the specific preset regex is to check for the existence of a `mod` or `module` token in the filename.